js/quote_tool.js

```diff
@@ -34,31 +192,127 @@
+  let total = 0;
+  let peopleAndNigth;
+  peopleAndNigth = form.numberOfPeople * form.numberOfNigth;
+  total += PRICENIGTH * peopleAndNigth;
+
+  total += form.transport.medNec ? BUSTRANS * form.numberOfPeople : 0;
+  total += form.transport.necCap ? BOATTRANS * form.numberOfPeople : 0;
+  total += form.transport.capNec ? BOATTRANS * form.numberOfPeople : 0;
+  total += form.transport.necMed ? BUSTRANS * form.numberOfPeople : 0;
+
+  total += form.food.breakFast ? BREAKFAST * peopleAndNigth : 0;
+  total += form.food.dinner ? DINNER * peopleAndNigth : 0;
+
+  total += form.tour.aguacatePlayasoledad ? TOURAPS * form.numberOfPeople : 0;
+  total += form.tour.sapzurroLamiel ? TOURSLM * form.numberOfPeople : 0;
+
+  total += form.hike.elCieloEltrebol ? HIKEECET * form.numberOfPeople : 0;
+  total += form.hike.elParaiso ? HIKEP * form.numberOfPeople : 0;
+  if (form.hike.laCoquerita.hike) {
+    total += HIKECO * form.numberOfPeople;
+    if (form.hike.laCoquerita.guide) {
+      total += HIKEGUI * form.numberOfPeople;
+    }
+  }
+  total += INSURANCE * peopleAndNigth;
+  return total;
+}
+
+/* call the elements of from */
+const submitData = () => {
+  cleanErrors();
+  let form = {};
+  form.transport = {};
+  form.food = {};
+  form.tour = {};
+  form.hike = {};
+  form.hike.laCoquerita = {};
+
+  form.numberOfPeople = document.getElementById("txtNumPeople").value;
+  form.numberOfNigth = document.getElementById("numNigth").value;
+
+  form.transport.medNec = document.getElementById("ts1").checked;
+  form.transport.necCap = document.getElementById("ts2").checked;
+  form.transport.capNec = document.getElementById("ts3").checked;
+  form.transport.necMed = document.getElementById("ts4").checked;
+
+  form.food.breakFast = document.getElementById("f1").checked;
+  form.food.dinner = document.getElementById("f2").checked;
+
+  form.tour.sapzurroLamiel = document.getElementById("t1").checked;
+  form.tour.aguacatePlayasoledad = document.getElementById("t2").checked;
+
+  form.hike.elCieloEltrebol = document.getElementById("h1").checked;
+  form.hike.elParaiso = document.getElementById("h2").checked;
+  form.hike.laCoquerita.hike = document.getElementById("h3").checked;
+
+  form.hike.laCoquerita.guide = document.getElementById("gu1").checked;
+
+  //calculate error
+  let errorList = validateData(form);
+  // Sends the collected data to the calculateCost funtion
+  let finalPrice = calculateCost(form);
+  // Print the information whit the total cost
+  document.getElementById("txtResult").innerHTML = finalPrice;
+
+  if (errorList) {
+    document.getElementById("txtErrors").appendChild(errorList);
+  }
+};
+
+// validate the opted data to verify the number of tours and nights.
+const validateData = (form) => {
+  var errors = [];
+  if (form.numberOfNigth < (getTrues(form.tour) + getTrues(form.hike))) {
+    errors.push(
+      "el numero seleccionado de dias es poco para la suma de tours y o caminatas, cada evento toma un dia."
```

Comment:
I reckon it sounds better if we say it like this:
"El número seleccionado de días es poco para la suma de tours y/o caminatas acumuladas, ten en cuenta que cada evento toma un día."